package.json

```diff
@@ -131,7 +131,7 @@
     "read-last-lines": "^1.1.0",
     "requirejs": "^2.3.1",
     "through2": "^2.0.3",
-    "uglify-js": "^2.7.5",
+    "uglify-js": "~2.7.5",
```

Comment:
We should probably open an issue in [`cwise`](https://github.com/scijs/cwise/issues/new) about this.